core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -700,7 +700,7 @@ public CostCalculator getEstimatedExchangesCostCalculator()
     }
 
     @Override
-    public TestingGroupProvider getGroupProvider()
```

Comment:
getGroupProviderManager?

Reply:
I've used the same "naming convention" as in:
```java
    @Override
    public TestingAccessControlManager getAccessControl()
    {
        return accessControl;
    }
```
Should I change both?

Reply:
No as `TestingAccessControlManager` is a `AccessControl`. Same with `GroupProvider`. So your code is good. Thanks.